javascript/javascript1/week1/nadiia/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Smart-ease projects</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+    <h1 class="main-text">Smart-ease project (for java-script practice)</h1>
+    
+    <section class="project-tile">
+    <script src="Age-ify.js"></script>
+    <h2 class="tile-title">Age-ify</h2>
+    <label name>Input year you were born</label>
+    <input type="number" id="yearOfBirth" class="input-bar">
+    <label>Input future year</label>
+    <input type="number" id="futureYear" class="input-bar">
+    <button onclick="AgeIfy()" class="result-button">Calculate future age</button>
+    <p id="futureAge"></p>
+    </section>
+
+    <section class="project-tile">
+    <script src="Goodboy-Oldboy.js"></script>
+    <h2 class="tile-title">Dogs variant of Age-Ify</h2>
+    <label>Input year your dog was born</label>
+    <input type="number" id="dogYearOfBirth" class="input-bar">
+    <label>Input future year</label>
+    <input type="number" id="dogsFutureYear" class="input-bar">
+    <button onclick="dogAgeIfy()" class="result-button">Calculate dog's future age in dog years</button>
+    <p id="dogFutureAge"></p>
+    </section>
+
+    <section class="project-tile">
+    <script src="Housey-pricey.js"></script>
+    <h2 class="tile-title">Housey pricey</h2>
+    <label>Input width of house(in meters)</label>
+    <input type="number" id="width" class="input-bar">
```

Comment:
Adding `for` attribute that matches the `id` in the `<input>` would associate it in a way, that when you click the label, the focus will immediately change to the input field. This is especially helpful for tiny clickable areas as in checkboxes and radio buttons, as then you don't need to be that careful and precise where you click. It can be quite an accessibility improvement. See if you can implement the same for the other `<label>` fields if you got the time.
```suggestion
    <label for="width">Input width of house(in meters)</label>
    <input type="number" id="width" class="input-bar">
```